data/microos/butane/script

```diff
@@ -4,25 +4,53 @@
 exec > >(exec tee -a /dev/console) 2>&1
 set -eux
 
-#
-### drives and filesystems
-#
-DRIVE='/dev/vdc'
-PART='/dev/vdc1'
+create_fs() {
+    DRIVE='/dev/vdc'
+    PART='/dev/vdc1'
+
+    # create partition and filesystem on additional drive
+    echo "label: gpt" | sfdisk "$DRIVE"
+    echo "name=testing_part" | sfdisk "$DRIVE" -N1
+
+    # label new partition for testing and create labeled ext4
+    mkfs.ext4 -L home "$PART"
+
+    # mount new partition
+    mount -t ext4 "$PART" /home
 
-# create partition and filesystem on additional drive
-echo "label: gpt" | sfdisk "$DRIVE"
-echo "name=testing_part" | sfdisk /dev/vdc -N1
+    # DEBUG
+    blkid
+    lsblk
+}
 
-# label new partition for testing and create labeled ext4
-mkfs.ext4 -L home "$PART"
+systemd_mount() {
+
+cat << EOF > /etc/systemd/system/home.mount
+[Unit]
+Before=local-fs.target
+Requires=systemd-fsck@dev-disk-by\x2dpartlabel-testing_part.service
+After=systemd-fsck@dev-disk-by\x2dpartlabel-testing_part.service
 
-# mount new partition
-mount -t ext4 "$PART" /home
+[Mount]
+Where=/home
+What=/dev/disk/by-partlabel/testing_part
+Type=ext4
 
-# DEBUG
-blkid
-lsblk
+[Install]
+RequiredBy=local-fs.target
+EOF
+
+}
+
+### set locale, keyboard and timezone
+# exception: sle-micro comes with symlink /etc/localtime, thus systemd-firstboot fails
+rm -f /etc/localtime
+systemd-firstboot --force --timezone=UTC --locale=en_US.UTF-8 --keymap=us
+echo 'FONT=eurlatgr.psfu' >> /etc/vconsole.conf
+
+if ! grep -q tumbleweed /etc/os-release; then
```

Comment:
IIRC this is because of missing `mkfs.ext4`, in which case this could just use btrfs instead

Reply:
Yup, the idea was to have pure ignition and pure combustion that will basically do the same. As the ignition's `config.fcc` is also used as the source of truth for verification, we would need another one that formats the testing drive as btrfs. I guess we can leave this as future improvement for now

Reply:
@Vogtinator as I would like to keep the script aligned with ignition config, are you ok, if the fork of ignition in order to create a test `btrfs` drive would be handled in a separate PR?

Reply:
https://progress.opensuse.org/issues/136931, this should be clarified